src/templates/event.html

```diff
@@ -29,7 +29,7 @@ <h1>{{ event.name|safe }}</h1>
                         <p>{{ event.description|safe }}</p>
                         {% if event.ticket_text %}
                         <div class="Event-register">
-                            <a href="{{ event.ticket_link }}" class="Button" target="_blank">{{ event.ticket_text|safe }}</a>
+                            <!-- <a href="{{ event.ticket_link }}" class="Button" target="_blank">{{ event.ticket_text|safe }}</a> -->
                         </div>
```

Comment:
Instead of commenting this line out, if you remove the `ticket_text` from `event.json`, it should take care of hiding the button (if statement on line 30)